### PR TITLE
  ABAP Logger Demonstration programs 

### PR DESCRIPTION
**Pull request by vidyadharg**
Demo program for [ABAP-Logger](https://github.com/ABAP-Logger/ABAP-Logger)

### CODE REVIEW

File: src/zdemo_logger04_no_tree.prog.abap [hunk 1]
```diff
@@ -0,0 +1,89 @@
+*&---------------------------------------------------------------------*
+*& Report zdemo_logger04_no_tree
+*&---------------------------------------------------------------------*
+*&
+*&---------------------------------------------------------------------*
+REPORT zdemo_logger04_no_tree MESSAGE-ID bl.
+
+DATA logger TYPE REF TO zif_logger.
```
Comment (vidyadharg) [inline src/zdemo_logger04_no_tree.prog.abap]: as zif_logger is not part of the repository.

**Comment by vidyadharg** [comment]
I have created PR for demos and should work with [ABAP-Logger-PR109](https://github.com/ABAP-Logger/ABAP-Logger/pull/109).

working first time with a linter. not sure about a few checks. feels they are false. Please review.

**Comment by vidyadharg** [comment]
as [PR 109](https://github.com/ABAP-Logger/ABAP-Logger/pull/119) is merged into [ABAP-logger](https://github.com/ABAP-Logger/ABAP-Logger) .  can somebody help to test DEMOs and approve them.

**Comment by mbtools** [comment]
Replaced by https://github.com/ABAP-Logger/demos/pull/8

@vidyadharg sorry that no one got to this earlier. There were conflicts with the naming and a few little issues due to changes in ABAP Logger. I created a new PR with your programs. Thanks for your contribution!

**Pull request closed without merging**